CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required (VERSION 2.8.12)
 project (PIO C Fortran)
-cmake_policy(VERSION 3.5.2)
+#cmake_policy(VERSION 3.5.2)
```

Comment:
Why do we want to have cmake_policy version higher than the min cmake version required to build PIO?